modules/eks/eks.tf

```diff
@@ -64,7 +64,7 @@ module "cluster" {
     for node_pool in local.parsed_node_pools :
     {
       name                          = lookup(node_pool, "name")
-      ami_id                        = lookup(node_pool, "ami_id")
+      ami_id                        = lookup(node_pool, "os")
```

Comment:
I have mixed feelings on calling ami_id == os, but I believe this is one of the downsides of having a common interface, right?